webapp/src/App.tsx

```diff
@@ -1,23 +1,31 @@
 import React, {useEffect} from 'react';
+import {Action, Store} from 'redux';
 import {useDispatch} from 'react-redux';
 
-import usePluginApi from 'hooks/usePluginApi';
+import {GlobalState} from 'mattermost-redux/types/store';
+
+import {RhsTitle} from 'components';
 
-//global styles
 import {pluginApiServiceConfigs} from 'constants/apiService.constant';
+import {defaultPage, defaultPerPage, pluginTitle} from 'constants/common.constants';
+import {iconUrl} from 'constants/illustrations.constants';
+
+import {Rhs} from 'containers';
+
 import useApiRequestCompletionState from 'hooks/useApiRequestCompletionState';
+import usePluginApi from 'hooks/usePluginApi';
 
 import {setConnected} from 'reducers/connectedState';
-import {defaultPage, defaultPerPage} from 'constants/common.constants';
 import {setIsRhsLoading} from 'reducers/spinner';
 
+// global styles
 import 'styles/main.scss';
 
 /**
  * This is the main App component for the plugin
  * @returns {JSX.Element}
  */
-const App = (): JSX.Element => {
+const App = ({registry, store}:{registry: PluginRegistry, store: Store<GlobalState, Action<Record<string, unknown>>>}): JSX.Element => {
```

Comment:
why `unknown`?

Reply:
This type was picked from file defining the `Plugin` class, which was taken from the plugin template